scripts/core/superdesk-api.d.ts

```diff
@@ -1218,6 +1218,10 @@ declare module 'superdesk-api' {
         editor3: {
             browserSpellCheck: boolean;
         };
+        pictures: {
```

Comment:
shouldn't this be optional? You can do it by adding a question mark `pictures?: {...`